web3/contract.py

```diff
@@ -1645,11 +2270,44 @@ def build_transaction_for_function(
         fn_kwargs=kwargs,
     )
 
-    prepared_transaction = fill_transaction_defaults(w3, prepared_transaction)
+    prepared_transaction = transactions.fill_transaction_defaults(w3, prepared_transaction)
```

Comment:
sort-of-duplicate-nitpick about importing the function instead of the whole file: 
```
from web3._utils.transactions import (
    fill_transaction_defaults,
)
```

Reply:
done in #2458